Classes/CharacterDefaults.gd

```diff
@@ -0,0 +1,316 @@
+class_name CharacterDefaults
+const script_name := "character_defaults"
+
+
+
+
+#### starting equipment ########################################################
+#       _             _   _                                _      
+#      | |           | | (_)                              (_)     
+#   ___| |_ __ _ _ __| |_ _ _ __   __ _    ___  __ _ _   _ _ _ __ 
+#  / __| __/ _` | '__| __| | '_ \ / _` |  / _ \/ _` | | | | | '_ \
+#  \__ \ || (_| | |  | |_| | | | | (_| | |  __/ (_| | |_| | | |_) |
+#  |___/\__\__,_|_|   \__|_|_| |_|\__, |  \___|\__, |\__,_|_| .__/
+#                                  __/ |          | |       | |    
+#                                 |___/           |_|       |_|     
+
+const starting_equipment := {
+	
+	
+	
+	
+	knight = [
+		"sword"
+	],
+	
+	
+	
+	
+	battle_mage = [
+		"staff"
+	],
+	
+	
+	
+	
+	berserker = [
+		"axe"
+	],
+	
+	
+	
+	
+	quick_shooter = [
+		"bow", "arrow", "dagger"
+	]
```

Comment:
Where are the starting skills/attacks saved

Reply:
Add a classattack array to add new attacks based of the chosen class, also the amount of arrows should be specified